qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -41,6 +43,22 @@
 logger = logging.getLogger(__name__)
 
 
+@bind_schema(BackendJobsLimitResponseSchema)
+class BackendJobsLimit(BaseModel):
+    """Jobs limit for a backend."""
+    def __init__(self, maximum_jobs: int, running_jobs: int, **kwargs: Any) -> None:
+        """Creates a new BackendJobsLimit instance.
+
+        Args:
+            maximum_jobs: maximum number of jobs for the backend.
+            running_jobs: current number of jobs running on the backend.
```

Comment:
Can you 1) check with Kevin (since he's working on the same thing for iqx) to ensure the field names match and 2) clarify a bit what `maximum_jobs` mean (e.g. `maximum number of jobs this account is allowed to run on this backend with this provider at a time`).